art/art.py

```diff
@@ -464,7 +467,7 @@ def __word2art(word, font, chr_ignore, letters, next_word):
     return result
 
 
-def text2art(text, font=DEFAULT_FONT, chr_ignore=True, decoration=None):
+def text2art(text, font=DEFAULT_FONT, show_font=False, chr_ignore=True, decoration=None):
```

Comment:
1. Add `show_font`  at the end of parameters (after `decoration`)
2. Update docstring